mobile/src/store/auth/reducer.ts

```diff
@@ -40,7 +40,7 @@ const reducer = createReducer(initialState, (builder) => {
   });
   builder.addCase(logout.fulfilled, (state) => {
     state.dataStatus = DataStatus.FULFILLED;
-    state.user = null;
+    state = initialState;
```

Comment:
As I remember it won't work. We can change state properties, but if we want to set state object we need to return a new one
```
return initialState;
```